src/jit/lowerxarch.cpp

```diff
@@ -4573,6 +4579,31 @@ void Lowering::SetMulOpCounts(GenTreePtr tree)
 }
 
 //------------------------------------------------------------------------------
+// SetContainsAVXFlags: Set ContainsAVX flag when it is floating type, set 
+// Contains256bitAVX flag when SIMD vector size is 32 bytes
+//
+// Arguments:
+//    isFloatingType    - true if it is floating type
+//    sizeOfSIMDVector  - SIMD Vector size
+//
+void Lowering::SetContainsAVXFlags(bool isFloatingType /* = true */, unsigned sizeOfSIMDVector /* = 0*/)
```

Comment:
>isFloatingTyp [](start = 40, length = 13)

Please rename this as isFloatingPointType.
(IsFloatingType sounds a bit strange!)